app/src/main/java/zapsolutions/zap/setup/SetupActivity.java

```diff
@@ -40,30 +37,20 @@ protected void onCreate(Bundle savedInstanceState) {
         }
 
 
-        // Set pin fragment as beginning fragment
-        showCreatePin();
+        // Set connection choice as beginning fragment
+        showConnectChoice();
```

Comment:
🤔  Is this called here and then in case of `FULL_SETUP` basically twice? ( https://github.com/LN-Zap/zap-android/pull/145/files#diff-c4b34cc8af17b9eb5903cc9312bf240aR45)